### PR TITLE
Render health/status check as an alert

### LINKED ISSUE #1652: Status checks do not render in React
- [ ] API for status checks
- [ ] render status checks

### PR DESCRIPTION
**Pull request by benvinegar**
Fixes #1652

### CODE REVIEW

**Comment by benvinegar** [comment]
Note that alerts are still rendering incorrectly, but @ckj said he'd take a look at that once this was in.

File: src/sentry/static/sentry/app/views/app.jsx [hunk 1]
```diff
@@ -37,6 +38,15 @@ var App = React.createClass({
         });
       }
     });
+
+    api.request('/internal/health/', {
+      success: (data) => {
+        if (data.problems) {
+          AlertActions.addAlert(data.problems.join(' '), 'error');
```
Comment (dcramer) [inline src/sentry/static/sentry/app/views/app.jsx]: thoughts about just rendering one alert per problem?

Comment (benvinegar) [inline src/sentry/static/sentry/app/views/app.jsx]: My thoughts are: it probably should render multiple alerts :) sec

**Comment by ckj** [comment]
Looks good, @benvinegar. Gonna merge it in.

**Pull request merged**